data/json/items/armor/jewelry.json

```
@@ -16,8 +16,8 @@
   {
     "id": "bead_ear",
     "type": "ARMOR",
-    "name": "beaded earring",
-    "description": "A simple bracelet of string and wooden beads.  You can wear it if you like, but it won't provide any effects.",
+    "name": "beaded earrings",
```

```suggestion
    "name": "pair of beaded earrings",
    "name_plural": "pairs of beaded earrings",
```

So it makes more sense when one has more than one pair of them. Same for the other names.

---

np